pkg/proto/runtime/v1/dapr_tracing.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtime
+
+import (
+	diagConsts "github.com/dapr/dapr/pkg/diagnostics/consts"
+)
+
+// This file contains additional, hand-written methods added to the generated objects.
+
+func (x *InvokeServiceRequest) AppendSpanAttributes(rpcMethod string, m map[string]string) {
```

Comment:
The `AppendSpanAttribute` methods must be hand-written in this file. It's not as nice as an approach as we have for HTTP, but it's consistent. The new test ensures that developers cannot add a message without the `AppendSpanAttribute` method

Again, the large number of `TODO`'s is due to the fact that these routes did not have proper tracing before. Completing this work should be done in separate PRs.